src/components/organisms/Footer/index.tsx

```diff
@@ -204,7 +179,7 @@ export const Footer = () => {
           >
             Go Conference
           </Typography>
-          <Typography fontSize={isTabletOrOver ? '12px' : '10px'}>
+          <Typography variant="caption" fontSize={isTabletOrOver ? '12px' : '10px'}>
```

Comment:
説明足りずにすみません。 `theme.ts` に以下のように caption variant に対してフォントサイズを設定してここでは varint の指定のみとするイメージでした 🙇 

```ts
caption: {
  fontSize: '12px',
  [breakpoints.down('sm')]: {
    fontSize: '10px',
  }
},

```

```suggestion
          <Typography variant="caption">
```

Reply:
なるほどです、ありがとうございます
修正します